Contest_Control_System_Requirements.md

```diff
@@ -274,20 +274,23 @@ on the contest network.
 
 ### Predefined Clarification Answers
 
-The CCS must be able to configure predefined answers to clarification
-requests (e.g., "No response, read problem statement." and "This will be
-answered after the practice session.") from the [ contest configuration
-file](#contestyaml), so that judges can choose to reply to a
-clarification request by selecting a predefined answer rather than being
-required to enter a specific answer. One of the predefined answers must
-be the "default" answer.
+The CCS must support predefined answers to clarification requests, so
+that judges can choose to reply to a clarification request by selecting
+a predefined answer rather than being required to enter a specific
+answer. At least the following predefined answers must be supported:
+
+* No comment, read problem statement.
+* This will be answered during the answers to questions session.
```

Comment:
I would suggest dropping the second and only requiring the first.

Reply:
Why? Isn't the second used more often than the first? I think it is the most used response during dress rehearsal / practice session and with that perhaps also the most used overall. What would dropping it do?

Reply:
> What would dropping it do?

Shorten the specification?

Maybe you're right...

Reply:
I like this last statement of yours